src/main/java/nl/tudelft/ti2206/group9/util/Base64Writer.java

```diff
@@ -0,0 +1,69 @@
+package nl.tudelft.ti2206.group9.util;
+
+import java.io.IOException;
+import java.io.UnsupportedEncodingException;
+import java.io.Writer;
+import java.util.Base64;
+
+/**
+ * This Writer decorates a Writer by encrypting files with Base64.
+ *
+ * @author Mathias
+ */
+public class Base64Writer extends Writer {
+
+    /** The Writer to be decorated with this Base64Writer. */
+    private final Writer writer;
+    /** The Base64 encoder. */
+    private final Base64.Encoder encoder = Base64.getEncoder();
+
+    /** Constructor which sets the Writer to be decorated.
+     * @param wrtr the Writer to be decorated
+     */
+    public Base64Writer(final Writer wrtr) {
+        super();
+        writer = wrtr;
+    }
+
+    /**
+     * Write an entire encoded String.
+     *
+     * The given input String is first encoded and then written to a file.
+     * The file is given by the internal Writer.
+     *
+     * @param input the String to be encoded and written
+     */
+    public void writeString(final String input) {
+        try {
+            final String encodedInput =
+                    encoder.encodeToString(input.getBytes("UTF-8"));
+            writer.write(encodedInput);
+        } catch (UnsupportedEncodingException e) {
+            GameObservable.OBSERVABLE.notify(GameObserver.Category.ERROR,
+                    GameObserver.Error.UNSUPPORTEDENCODINGEXCEPTION,
+                    "Base64Writer.writeString()", e.getMessage());
+        } catch (IOException e) {
+            GameObservable.OBSERVABLE.notify(GameObserver.Category.ERROR,
+                    GameObserver.Error.IOEXCEPTION,
+                    "Bas64Writer.writeString(String)", e.getMessage());
+        }
+    }
+
+    @Override
+    public void write(final char[] cbuf, final int off,
```

Comment:
Is this method (and the overriden read method in Reader) tested?

<review, testing